packages/cdktf-cli/lib/get/generator/constants/provider-namespaces.ts

```diff
@@ -0,0 +1,42 @@
+export type ResourceNamespace = { name: string; comment: string };
+
+// Namespace to Resource mapping
+const aws = {
+  ACM: ["acm_certificate", "acm_certificate_validation"],
+};
+
+const PROVIDER_NAMESPACES: Record<string, Record<string, string[]>> = {
+  aws,
+};
+
+// Comments per namespace mapping
+const awsNamespaceComments: { [key in keyof typeof aws]: string } = {
+  ACM: "AWS Certificate Manager",
+};
+const PROVIDER_NAMESPACES_COMMENT: {
+  [key in keyof typeof PROVIDER_NAMESPACES]: Record<string, string>;
+} = {
+  aws: awsNamespaceComments,
+};
+
+export function getResourceNamespace(
```

Comment:
That would be usable in other contexts as well, right? Like in cdktf convert

Reply:
Yeah, I'd pull it out into convert (or maybe another shared package) once we settled that this is the right way